pentaho-js/src/test/javascript/cdf/components/TableComponent-spec.js

```diff
@@ -196,5 +196,27 @@ define([
       dashboard.update(tableComponent);
     });
 
+    /**
+     * ## The Table Component # can update when server side is true
+     */
+    it("can update when server side is true", function(done) {
```

Comment:
In addition this test takes into account all the component if for any reason another bug does not let the component render when the flag is up  (this problem happened before) this will warn us, if a more "specific" test is made we can be sure that the new code is fine but not necessarily the whole